packages/slice-machine/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -1,6 +1,24 @@
 import { Text } from "@prismicio/editor-ui";
-import { FC, PropsWithChildren } from "react";
+import { FC } from "react";
 
-export const Breadcrumb: FC<PropsWithChildren> = ({ children }) => {
-  return <Text color="grey11">{children}</Text>;
+import * as styles from "./Breadcrumb.css";
+
+export const Breadcrumb: FC<{
```

Comment:
Maybe we could export the props in BreadcrumbProps type to have consistency with code guidelines?